README.md

```diff
@@ -85,24 +85,24 @@ tfswitch #will automatically switch to terraform version 0.14.4
 3. Hit **Enter** to install.
 ### Install latest implicit version for stable releases
 1. Install the latest implicit stable version.
-2. Ex: `tfswitch -s 0.13` or `tfswitch --latest-stable 0.13` downloads 0.13.6 (latest) version.
+2. Ex: `tfswitch -s 0.13` or `tfswitch --latest-stable 0.13` downloads <1.1.0, >=1.0.0, 1.0 downloads <2.0.0, >=1.0.0.
 3. Hit **Enter** to install.
 ### Install latest implicit version for beta, alpha and release candidates(rc)
 1. Install the latest implicit pre-release version.
-2. Ex: `tfswitch -p 0.13` or `tfswitch --latest-pre 0.13` downloads 0.13.0-rc1 (latest) version.
+2. Ex: `tfswitch -p 0.13` or `tfswitch --latest-pre 0.13` downloads 0.13.0-rc1.
```

Comment:
This is a bit confusing that `-p` works differently than e.g. `-s`. Though this flag's audience assumably is quite small to rework the logic 🤔

Reply:
Yeah, `-p` only works for `major.minor` while `-s` works for   `major.minor.patch`.

At that time, I do not know modify the regex to work with `0.0.X` with `X` being `0-rc1` (alphanumeral+dash).

I am open to modifications, if anyone can make it better.